python/cudf/cudf/tests/test_column.py

```diff
@@ -29,8 +29,37 @@
 
 @pytest.fixture(params=dtypes, ids=dtypes)
 def pandas_input(request):
```

Comment:
This fixture was throwing a lot of warnings about casting -- in particular, things like creating a Series with dtype `int8` from data that could exceed 127 (the max value was 1000). Instead, I implemented a function that creates a few random values based on the type.